libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/DefaultTracer.java

```diff
@@ -44,13 +45,13 @@ public SpanScope startSpan(String spanName) {
 
     @Override
     public SpanScope startSpan(String spanName, Attributes attributes) {
-        return startSpan(spanName, null, attributes);
+        return startSpan(spanName, SpanContext.EMPTY, attributes);
```

Comment:
```suggestion
        return startSpan(spanName, null, attributes);
```

I there is no parent, should be no context

Reply:
This is conflicting/not resolving with other overloaded method public SpanScope startSpan(String spanName, HttpHeader header, Attributes attributes); for null value.

Reply:
I think this is fine (for now), the caller would never pass null for `headers` (empty collection instead)